src/index.ts

```diff
@@ -21,6 +25,10 @@ export const defaultConfig = {
   disableGen: false,
 };
 
+/**
+ * Class to represent a TestxServer.
+ * @class
```

Comment:
We don't need to add tags like `@class`, `@constructor`, `@sync`. They where needed when javascript didn't have any classes and functions were used for that scope, same for sync

Reply:
I get it, thanks :smile: